Modules/Utility/os.lua

```diff
@@ -1,5 +1,14 @@
 -- @author Narrev
 
+-- Necessary string tables
+local dayNames = {[0] = "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"}
+local months = {"January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"}
+
+-- Localize functions
+local floor, sub, find, gsub, format = math.floor, string.sub, string.find, string.gsub, string.format
+local time = os.time
+local type, wait, tick, tonumber, assert = type, wait, tick, tonumber, assert
+
 local function date(formatString, unix)
```

Comment:
Might make sense to rename `unix` to be `unixTime` (we probably don't want to refactor the whole class to match the wiki standards at this point, until those are better defined) (someday?).

I would